DogTales/src/player.hpp

```diff
@@ -1,10 +1,34 @@
 #pragma once
 #include <bave/app.hpp>
-#include <bave/graphics/shape.hpp>
+//#include <bave/graphics/shape.hpp>
+#include <bave/graphics/sprite.hpp>
+
+enum class Direction {
+	UP,
+	DOWN,
+	LEFT,
+	RIGHT
+};
+
+constexpr glm::vec2 PLAYER_SPEED { 500.0f, 500.0f };
```

Comment:
Screaming case is generally reserved for macros. For constants, the stdlib uses `foo_v` (for `value`), which I also use. Enums can get tricky if the value collides with a language keyword (eg `and`), most use `PascalCase` with an `e` or `k` prefix to avoid that.

Reply:
Makes sense, I will keep to that convention moving forward!